pkg/playoutdelay/playout_delay.go

```diff
@@ -0,0 +1,52 @@
+// Package playoutdelay implements the playout delay header extension.
+package playoutdelay
+
+import (
+	"encoding/binary"
+	"errors"
+)
+
+const (
+	// transport-wide sequence
+	playoutDelayExtensionSize = 3
+	playoutDelayMaxValue      = (1 << 12) - 1
+)
+
+var (
+	errPlayoutDelayInvalidValue = errors.New("invalid playout delay value")
+	errTooSmall                 = errors.New("playout delay header extension too short")
+)
+
+// Extension is a extension payload format in
+// https://webrtc.googlesource.com/src/+/refs/heads/main/docs/native-code/rtp-hdrext/playout-delay
+// 0                   1                   2                   3
+// 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+// +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+// |  ID   | len=2 |       MIN delay       |       MAX delay       |
+// +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+type Extension struct {
```

Comment:
I think this should be part of pion/rtp

Reply:
Created https://github.com/pion/rtp/pull/196 to add it to pion/rtp.